recipes/create-bindinng-with-optional-arguments.md

```diff
@@ -0,0 +1,43 @@
+# Create binding with optionnal arguments
```

Comment:
Could simply be `Bind optional arguments`

Reply:
updated.